packages/front/src/components/Claim/ClaimCreate.jsx

```diff
@@ -3,18 +3,20 @@ import axios from "axios";
 import { Editor, EditorState } from "draft-js";
 import { stateToHTML } from "draft-js-export-html";
 import "draft-js/dist/Draft.css";
-import { Typography, Form, Input, Button } from "antd";
+import { Typography, Form, Input, Button, message, Row, Col } from "antd";
 
 const { Title } = Typography;
 class ClaimCreate extends Component {
+    formRef = React.createRef();
```

Comment:
@thesocialdev shouldn't we start 'typing' those new variables?

Reply:
@wellcda definitely, the next commits will update that.